src/collections/blog/Blog.style.js

```diff
@@ -207,6 +207,52 @@ export const BlogWrapper = styled.div`
     border-bottom: 1px dashed ${(props) => props.theme.primaryLightColor};
   }
 
+
+  /* ==============
+styles for figure with caption
+
+Example: 
+// <figure class="imgWithCaption fig-right">
+//   <img src="" />
+//   <figcaption>
+//     <p>This is text for the image</p>
+//   </figcaption>
+// </figure>
+=======================*/
+
+  figure.imgWithCaption {
+    margin-bottom: 1rem;
+    box-shadow: 0px 5px 10px 1px rgba(0, 179, 159, 0.5);
```

Comment:
Dont hard code the colors please

Reply:
This uses a reduced opacity so I cannot use the theme color just yet. We were discussing modifying the themes file to consider opacity as well but that's not implemented yet.